### ISSUE TITLE
Link is not clickable

### ISSUE THREAD

**Issue by sabuj073** [munafio/chatify-demo#41]
Links are not clickable when sent

**Comment by munafio** [comment]
@sabuj073 Yeah, its not supported .. its up to you, you can customize it to accept links (convert links in string a link tag)
